src/main/java/ro/hiringsystem/service/impl/JobServiceImpl.java

```diff
@@ -59,9 +61,26 @@ public void removeElementById(UUID id) {
     @Override
     public void saveElement(JobDto jobDto) {
         Job job = jobMapper.toEntity(jobDto);
-        jobRepository.save(jobMapper.toEntity(jobDto));
```

Comment:
Why did you separate updating from inserting? I think .save() would do exactly what you have done automatically

Reply:
You're right. I double checked it now and .save() works just fine.